ParallaxGenLib/src/BethesdaDirectory.cpp

```diff
@@ -157,8 +157,8 @@ auto BethesdaDirectory::getFile(const filesystem::path &RelPath, const bool &Cac
     const bsa::tes4::version BSAVersion = BSAStruct->Version;
     const bsa::tes4::archive BSAObj = BSAStruct->Archive;
 
-    string ParentPath = wstrToStr(RelPath.parent_path().wstring());
-    string Filename = wstrToStr(RelPath.filename().wstring());
+    string ParentPath = UTF16toWindows1252(RelPath.parent_path().wstring());
```

Comment:
The BSAs in [this mod](https://www.nexusmods.com/skyrimspecialedition/mods/58403) still throw errors with this encoding

From the author of the BSA library when I brought up the errors from the above mod:
```
You're looking at [mojibake](https://en.wikipedia.org/wiki/Mojibake). The real string is meshes\syerscote\goldpot01苃냂.nif, which google translate tells me is korean. This string is a mixture of ascii and utf-16, but most software can't pick up on that and guesses [windows-1252](https://en.wikipedia.org/wiki/Windows-1252) instead. This library doesn't make any guarantees about encoding specifically because nonsense like this can happen, and the game will happily accept it.
```

So it seems like utf16? But the operator looks like it only takes strings, so we probably have to put the UTF16 string's bytes into a normal string?

Reply:
When testing I found that too. The problematic filenames contain the unicode character `U+201a`, a comma. Though the character is defined in ANSI `0x82`, the BSA library has a bug wher files with ANSI values between `0x70` and `0x8F` can't be retrieved, I reported it here:

https://github.com/Ryan-rsm-McKenzie/bsa/issues/17